pages/home/arcade-cli.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: "Arcade CLI"
+description: "Learn how to install and use the Arcade CLI"
+---
+
+import { Tabs } from "nextra/components";
+
+# The Arcade CLI
+
+The Arcade CLI is a command-line tool that allows you to manage your Arcade deployments, generate, test, and manage your toolkits, and more.
+
+## Installation
+
+Like all python packages, the Arcade CLI needs to be installed within the python virtual environment you are using for your Arcade deployment.
+
+<Tabs items={["Standard Python", "Python + Conda", "Python + uv"]}>
+<Tabs.Tab>
+```bash
+python -m venv .venv
+source .venv/bin/activate
+```
+</Tabs.Tab>
+<Tabs.Tab>
+```bash
+# install miniconda: https://www.anaconda.com/docs/getting-started/miniconda/install
+
+conda create --name arcade
+conda activate arcade
+```
+</Tabs.Tab>
+<Tabs.Tab>
+```bash
+# install uv: https://docs.astral.sh/uv/getting-started/installation/
+
+uv venv
+uv source .venv/bin/activate
+```
+</Tabs.Tab>
+</Tabs>
+
+Now that your python virtual environment is activated, you can install the Arcade CLI with the following command:
+```bash
+pip install arcade-ai
+```
+
+## Usage
+
+```bash
+$ arcade --help
```

Comment:
probably fine for now. I would like to graduate to something like: https://ngrok.com/docs/agent/cli/

OR we could have `--help` be the cannonical place. Mostly worried about things drifting.

Reply:
For now, I'll list all the command's help for SEO I guess